### PR TITLE
docs(grammatical): correct typos and improve grammar

### PR DESCRIPTION
**Pull request by AbhineshJha**
This pull request addresses and corrects grammatical and spelling errors throughout the codebase to improve code readability.
The changes include fixing typos, punctuation, and ensuring proper sentence structure without introducing functional modifications.
These updates are intended to enhance the quality of documentation and code comments.
The focus is on making the codebase more polished and user-friendly by eliminating language-related issues.

### CODE REVIEW

**Comment by AbhineshJha** [comment]
hey @alandtse ! please review

**Comment by alandtse** [comment]
Please don't ping me. I review when I have time.

**Comment by AbhineshJha** [comment]
okay ! sir

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -50,8 +50,7 @@ Use [black](https://github.com/ambv/black) to make sure the code follows the sty
 
 This custom integration is based on [integration_blueprint template](https://github.com/custom-components/integration_blueprint).
 
-It comes with development environment in a container, easy to launch
-if you use Visual Studio Code. With this container you will have a stand alone
+It comes with development environment in a container that is easy to launch if you use Visual Studio Code. With this container you will have a stand alone.
```
Comment (alandtse) [inline CONTRIBUTING.md:53]: Last sentence is a fragment. Remove period.

Comment (AbhineshJha) [reply CONTRIBUTING.md]: done sir

**Comment by AbhineshJha** [comment]
sir ! @alandtse  you can check now it's better.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -50,8 +50,8 @@ Use [black](https://github.com/ambv/black) to make sure the code follows the sty
 
 This custom integration is based on [integration_blueprint template](https://github.com/custom-components/integration_blueprint).
 
-It comes with development environment in a container, easy to launch
-if you use Visual Studio Code. With this container you will have a stand alone
+It comes with development environment in a container that is easy to launch
+if you use Visual Studio Code. With this container you will have a stand alone.
```
Comment (alandtse) [inline CONTRIBUTING.md:54]: The period at the end of "stand alone." is incorrect. Please fix that so we don't introduce an error with this correction.

**Pull request closed without merging**